### DISCUSSION TITLE
Paid but not subscribed.

### DISCUSSION THREAD

**Discussion by mahaonanStart** [community/community#65544] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

My PayPal account shows that it automatically renewed my Copilot subscription, but my Copilot account is not activated. I forgot to cancel the auto-renewal, originally I didn’t want to renew it, but now it has automatically deducted the fee without activating it.
What happened?
<img width="782" alt="image" src="https://github.com/community/community/assets/44892851/d373d796-0246-4638-abbe-7612d3d07a39">
<img width="827" alt="image" src="https://github.com/community/community/assets/44892851/9ea132e8-e291-4d74-8bdd-d2259e68eb13">

**Comment by mdtanvirahamedshanto** [comment]
sent a mail your problem

**Reply by mahaonanStart** [reply]
Who should I email?

**Comment by davecheney** [comment, marked as answer]
@mahaonanStart your payment processor declined the charge. please contact support https://support.github.com/contact.
